### PR TITLE
[CSL-1617] Remove lwallet and add auxx. Fix benchmarks

### PR DESCRIPTION
**Pull request by pva701**

### CODE REVIEW

File: scripts/bench/buildbench.sh [hunk 1]
```diff
@@ -3,4 +3,4 @@
 # dev-mode : allow to generate custom genesis block
 # CONFIG=.. selects section from config file (core/constants.yaml)
 
-stack build --flag cardano-sl-core:dev-mode --flag cardano-sl-core:-asserts cardano-sl cardano-sl-lwallet cardano-sl-wallet cardano-sl-explorer
+stack build --flag cardano-sl-core:dev-mode --ghc-options=-DCONFIG=benchmark --flag cardano-sl-core:-asserts cardano-sl cardano-sl-auxx cardano-sl-wallet cardano-sl-explorer
```
Comment (dcoutts) [inline scripts/bench/buildbench.sh]: I think `--ghc-options=-DCONFIG=benchmark` was removed. Please check.

Comment (pva701) [reply scripts/bench/buildbench.sh]: you are right

Comment (pva701) [reply scripts/bench/buildbench.sh]: and I removed it here https://github.com/input-output-hk/cardano-sl/pull/1604/commits/775234f319651b71f8cade74d62e5c49c2c0e7cc#diff-c49512cad725a697f559387fda657638R6
@dcoutts

Comment (dcoutts) [reply scripts/bench/buildbench.sh]: Ah, so you did. Thanks.

**Pull request merged**